### PR TITLE
Upload GRR image to GitHub container registry instead of DockerHub.

### PR DESCRIPTION
**Pull request by mbushkov**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -259,13 +259,14 @@ jobs:
           export BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)
           ./appveyor/docker_build/build_docker_image.sh
       - if: ${{ github.event_name == 'push' }}
-        name: Log in to Docker Hub
+        name: Login with Github Container registry
```
Comment (panhania) [inline .github/workflows/build.yml:262]: Nit: `GitHub`.

Also, from what I see, it is called "GitHub Packages", not "GitHub Container".

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -259,13 +259,14 @@ jobs:
...
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       - if: ${{ github.event_name == 'push' }}
-        name: Push to ${{ env.DOCKER_REPOSITORY }}
+        name: Push to GitHub Container registry
```
Comment (panhania) [inline .github/workflows/build.yml:269]: Ditto.

**Review by panhania — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (mbushkov) [reply .github/workflows/build.yml:262]: I humbly disagree: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

Comment (mbushkov) [reply .github/workflows/build.yml]: Fixed Github -> GitHub.

File: .github/workflows/build.yml (hunk 2 above)
Comment (mbushkov) [reply .github/workflows/build.yml:269]: As said above, I think `GitHub container registry` is a valid term (this is what `ghcr.io` stands for) and what's used in the documentation.

**Review by panhania — APPROVED** [review]

**Pull request merged**